unit-tests/resources/commcare-apps/archive_form_tests/modules-0/forms-0.xml

```diff
@@ -0,0 +1,25 @@
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
```

Comment:
I know @ctsims is in favor of making these test resources as lean as possible. I'm personally very bad at determining what exactly can and can't be removed, but this looks like it probably has a bunch of stuff in it that isn't needed